Patch-Source: https://sources.debian.org/data/main/libt/libtest-unit-perl/0.25-7/debian/patches/30_fix-xism-test-suite-failures.patch

Description: Fix test-suite failures due to ?-xism vs ?^
 ?^ was introduced with Perl 5.14 as abbreviation for ?d-imsx
Author: Axel Beckert <abe@debian.org>
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=62771

Index: libtest-unit-perl/t/tlib/AssertTest.pm
===================================================================
--- libtest-unit-perl.orig/t/tlib/AssertTest.pm	2013-12-26 01:17:02.338121179 +0100
+++ libtest-unit-perl/t/tlib/AssertTest.pm	2013-12-26 01:17:50.658363830 +0100
@@ -70,7 +70,7 @@
 
         'bang'  => [ __LINE__, sub { shift->assert(0, 'bang')              } ],
         'bang'  => [ __LINE__, sub { shift->assert('', 'bang')             } ],
-        "'qux' did not match /(?-xism:foo)/"
+        "'qux' did not match /(?^:foo)/"
                 => [ __LINE__, sub { shift->assert(qr/foo/, 'qux')         } ],
         'bang'  => [ __LINE__, sub { shift->assert(qr/foo/, 'qux', 'bang') } ],
         'a ne b'=> [ __LINE__, sub { shift->assert($coderef, 'a', 'b')     } ],
@@ -253,7 +253,7 @@
         q{expected '', got 'foo'}           => [ 'foo',  ''      ], 
         q{expected 'foo', got ''}           => [ '',     'foo'   ], 
         q{expected 5, got 4}                => [ $adder, 5       ], 
-        q{'foo' did not match /(?-xism:x)/} => [ 'foo',  qr/x/   ], 
+        q{'foo' did not match /(?^:x)/} => [ 'foo',  qr/x/   ], 
     );
     my @tests = ();
     while (@checks) {
